profile/README.md

```diff
@@ -0,0 +1,71 @@
+# Welcome to Xen-Troops!
+
+We develop embedded automotive Xen for different hardware platforms and actively contribute to Xen, Linux, OPTEE, Zephyr, and other communities.
```

Comment:
"other open source projects". I can't say that we contribute to communities themselves :)

Reply:
Fixed.